### PR TITLE
Silent Auth

### PR DESCRIPTION
**Pull request by digvijay-otpless**
* android sdk update to 2.3.3 and ios sdk updated to 2.1.3
* web inspectable feature added for ios

### CODE REVIEW

File: example/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -40,7 +41,7 @@
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data
                     android:host="otpless"
-                    android:scheme= "otpless.your_appid_in_smallcase"/>
+                    android:scheme= "otpless.5e62zcanetd9urnxpz80"/>
```
Comment (digvijay-otpless) [inline example/android/app/src/main/AndroidManifest.xml]: remove

File: example/ios/Runner/Info.plist [hunk 2]
```diff
@@ -31,7 +31,7 @@
 			<string>otpless</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>otpless.your_appid_in_smallcase</string>
+				<string>otpless.5e62zcanetd9urnxpz80</string>
```
Comment (digvijay-otpless) [inline example/ios/Runner/Info.plist]: remove

File: example/lib/main.dart [hunk 3]
```diff
@@ -25,43 +26,48 @@ class _MyAppState extends State<MyApp> {
...
   String otp = '';
+  bool isInitIos = false;
 
   @override
   void initState() {
     super.initState();
-    _otplessFlutterPlugin.initHeadless("YOUR_APPID");
-    _otplessFlutterPlugin.setHeadlessCallback(onHeadlessResult);
+    if (Platform.isAndroid) {
+      _otplessFlutterPlugin.initHeadless("5e62zcanetd9urnxpz80");
```
Comment (sparsh-otpless) [inline example/lib/main.dart]: remove appid

File: example/lib/main.dart [hunk 4]
```diff
@@ -25,43 +26,48 @@ class _MyAppState extends State<MyApp> {
...
-      var message = "";
-      if (result['data'] != null) {
-        final token = result['data']['token'];
-        message = "token: $token";
-      }
-      setState(() {
-        _dataResponse = message ?? "Unknown";
-      });
-    }, arg);
+    Map<String, dynamic> arg = {'appId': "5e62zcanetd9urnxpz80"};
```
Comment (sparsh-otpless) [inline example/lib/main.dart]: remove appid

File: example/ios/Runner/Info.plist (hunk 2 above)
Comment (sparsh-otpless) [inline example/ios/Runner/Info.plist]: remove appid

File: example/lib/main.dart [hunk 5]
```diff
@@ -25,43 +26,48 @@ class _MyAppState extends State<MyApp> {
...
+    _otplessFlutterPlugin.openLoginPage(onHeadlessResult, arg);
   }
 
   Future<void> startHeadlessWithWhatsapp() async {
+    if (Platform.isIOS && !isInitIos) {
+      _otplessFlutterPlugin.initHeadless("5e62zcanetd9urnxpz80");
```
Comment (sparsh-otpless) [inline example/lib/main.dart]: remove appid

File: example/lib/main.dart [hunk 6]
```diff
@@ -25,43 +26,48 @@ class _MyAppState extends State<MyApp> {
...
-      setState(() {
-        _dataResponse = jsonEncode(result);
-      });
-    }, arg);
+    _otplessFlutterPlugin.startHeadless(onHeadlessResult, arg);
   }
 
   Future<void> startHeadlessForPhoneAndEmail() async {
+    if (Platform.isIOS && !isInitIos) {
+      _otplessFlutterPlugin.initHeadless("5e62zcanetd9urnxpz80");
```
Comment (sparsh-otpless) [inline example/lib/main.dart]: remove appid

**Review by sparsh-otpless — CHANGES_REQUESTED** [review]
checkout the comments

**Review by dev-otpless-tech — APPROVED** [review]

**Pull request merged**